parity/url.rs

```diff
@@ -16,33 +16,20 @@
 
 //! Cross-platform open url in default browser
 
-#[cfg(windows)]
-mod shell {
-	extern crate winapi;
-
-	use self::winapi::*;
-	extern "system" {
-		pub fn ShellExecuteA(
-			hwnd: HWND, lpOperation: LPCSTR, lpFile: LPCSTR, lpParameters: LPCSTR, lpDirectory: LPCSTR,
-			nShowCmd: c_int
-		) -> HINSTANCE;
-	}
-
-	pub use self::winapi::SW_SHOWNORMAL as Normal;
-}
-
 #[cfg(windows)]
 pub fn open(url: &str) {
 	use std::ffi::CString;
 	use std::ptr;
+	use winapi::um::shellapi::ShellExecuteA;
+	use winapi::um::winuser::SW_SHOWNORMAL as Normal;
 
 	unsafe {
```

Comment:
I think you can remove unsafe here now!

Reply:
Can you explain me why this can be removed? `ShellExecuteA` is still marked as unsafe.

Reply:
ah, for some reason I thought that winapi interface is safe... I should have double checked :/

Reply:
But shouldn't the build have failed?

Reply:
@andresilva I think CI only runs on Linux right? So, using conditional compilation flags will never compile stuff that isn't `#[cfg(target_os="linux")]`

Reply:
You're right didn't think of that 👍